btk/draw_blends.py

```diff
@@ -69,7 +69,7 @@ def get_catsim_galaxy(entry, filt, survey, no_disk=False, no_bulge=False, no_agn
         Galsim galaxy profile
     """
     components = []
-    total_flux = mag2counts(entry[filt.name + "_ab"], survey.name, filt.name).to("electron").value
+    total_flux = mag2counts(entry[filt.name + "_ab"], survey, filt).to("electron").value
```

Comment:
```suggestion
    total_flux = mag2counts(entry[filt.name + "_ab"], survey, filt).to_value("electron")
```

Concerning the fact the input mag seems to be an AB magnitude, is this a residual from the past or not. Because I though we had changed this and defaulted to classical magnitudes.